### ISSUE TITLE
[Bug] 2.4.7 MS Edge (Chromium) will cause Netflix Error UI3013

### ISSUE THREAD

**Issue by amanteae** [polywock/globalSpeed#3]
Global Speed 2.4.7 MS Edge (Chromium  80.0.361.62) will cause Netflix Error UI3013. Only way to fix it is to uninstall or disable G.S (not suspend).

**Comment by polywock** [comment]
Sorry about that. Good news, I have already noticed this bug and pushed a fix. It's live on Chrome and Firefox, but not yet on Edge. I pushed the update two days ago, so I expect it go live today or tomorrow. 

For context on why this bug exists. I recently added support for unconnected media elements. That means support for Spotify, Amazon Music, NPR, and many other music and podcast streaming sites. The original way I implemented it was destructive (at least for Netflix). I found a cleaner way to support unconnected elements.
